lib/s3-service.js

```diff
@@ -7,7 +7,39 @@
  */
 
 // Require AWS SDK
-const AWS = require('aws-sdk'); // AWS SDK
+const { S3Client, GetObjectCommand } = require('@aws-sdk/client-s3'); // AWS SDK
+const { getSignedUrl } = require('@aws-sdk/s3-request-presigner');
 
 // Export
-module.exports = new AWS.S3();
+exports.client = new S3Client();
+
+exports.getObject = (params) => {
+  const cmd = new GetObjectCommand(params);
+  const promise = this.client.send(cmd);
+  return {
+    promise: () => promise,
+  };
+};
+
+exports.getSignedUrl = async (
+  type,
+  { Expires, ...params },
+  callback = () => {}
+) => {
+  let command;
+  switch (type) {
+    case 'getObject':
+      command = new GetObjectCommand(params);
+      break;
```

Comment:
What about putObject?

Reply:
Am I missing something, where is putObject used in this library?

Reply:
my bad, you're right.
Let's just make sure CI passes before we merge this in